src/main/java/se/iths/crimedatabase/entity/Criminal.java

```diff
@@ -0,0 +1,43 @@
+package se.iths.crimedatabase.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Criminal {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String firstName;
+    private String lastName;
+
```

Comment:
I suggest we add: LocalDate dateOfBirth as a field here

Reply:
Fine by me!